InnerEye/ML/configs/other/HelloContainer.py

```diff
@@ -30,16 +31,15 @@ class HelloDataset(Dataset):
     # x = torch.rand((N, 1)) * 10
     # y = 0.2 * x + 0.1 * torch.randn(x.size())
     # xy = torch.cat((x, y), dim=1)
-    # np.savetxt("InnerEye/ML/configs/other/hellocontainer.csv", xy.numpy(), delimiter=",")
-    def __init__(self, root_folder: Path, start_index: int, end_index: int) -> None:
+    # np.savetxt("Tests/ML/test_data/hellocontainer.csv", xy.numpy(), delimiter=",")
```

Comment:
hellocontainer.csv is now in InnerEye/ML/configs/other/

Reply:
Good catch, Anton moved that as part of his fixes earlier this week. Ill fix that.

Reply:
Done